components/d2l-activity-list-item/d2l-activity-list-item.js

```
@@ -232,6 +251,10 @@ class D2lActivityListItem extends mixinBehaviors([IronResizableBehavior, D2L.Pol
 				type: String,
 				value: ''
 			},
+			actionEnrollHide: {
```

cleanup, default is to hide the enrollment button.